net.crossfireatlas.crossfire.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: net.crossfireatlas.crossfire
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: org.freedesktop.Sdk.Extension.openjdk11
+command: net.crossfireatlas.crossfire.start.sh
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: 
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: Crossfire_client
+    buildsystem: simple
+    build-commands: 
+      - install -D download /app/bin/jxclient.jar
+      - install -D net.crossfireatlas.crossfire.start.sh /app/bin/net.crossfireatlas.crossfire.start.sh
+      - install -D net.crossfireatlas.crossfire.desktop /app/share/applications/net.crossfireatlas.crossfire.desktop
+      - install -D net.crossfireatlas.crossfire.appdata.xml /app/share/metainfo/net.crossfireatlas.crossfire.appdata.xml
+    sources:
+      - type: file
+        url: https://sourceforge.net/projects/crossfire/files/jxclient/jxclient.jar/download
+        sha256: e6af744ea986da0991233787be6c9bc5535a7f3fe8ea682fe3284f1152e67b77
+      - type: file
+        path: net.crossfireatlas.crossfire.start.sh
```

Comment:
the files installed above still are not referenced here. .desktop, .appdata.xml and icon in the right format.

This is the build failure atm.

Reply:
OK, done.